File: packages/targets/sources/ios/bindings/Color.swift
extension Color {
    /**
     A `UIColor` representation of the color.
     */
    @objc public var color: UIColor {
        let brightness = l + s * min(l, 1 - l)
        let saturation = (brightness == 0) ? 0 : 2 - 2 * l / brightness
        return UIColor(hue: h, saturation: saturation, brightness: brightness, alpha: a)
    }
}

File: src/compiler/targets/test/goldens/Primitives/diez-target-test-stub-android/src/main/java/org/diez/targetTestStub/Color.kt
package org.diez.targetTestStub

/**
 * A component encapsulating color, including alpha transparency.
 * 
 * You can use the provided static constructors [[Color.rgb]], [[Color.rgba]], [[Color.hsl]], [[Color.hsla]], and
 * [[Color.hex]] to conveniently create color primitives using familiar patterns for color specification.
 *
 */
data class Color(
    /**
     * Provides simple hue-saturation-lightness-alpha color data.
     *
     * 0
     */
    val h: Float,
    /**
     * Provides simple hue-saturation-lightness-alpha color data.
     *
     * 0
     */
    val s: Float,
    /**
     * Provides simple hue-saturation-lightness-alpha color data.
     *
     * 0
     */
    val l: Float,
    /**
     * Provides simple hue-saturation-lightness-alpha color data.
     *
     * 1
     */
    val a: Float
) {
    companion object {}
}
